tests/no-internal.js

```diff
@@ -137,7 +137,56 @@ ruleTester.run(
           { message: 'function "internal" is internal.' },
           { message: 'method "Public.internalMethod" is internal.' }
         ]
-      }
+      },
+      {
+        // itwin scope with multiple usages
```

Comment:
nit: instead of new tests, you can add these multi-usages to the previous tests. It's a separate concept but I think an unnecessary amount of tests with overlapping differences

Reply:
Yes, I agree